1/src/_backend/index.js

```diff
@@ -1,12 +1,26 @@
 const { ApolloServer } = require('apollo-server');
-const typeDefs = require('./schema');
-const resolvers = require('./resolvers');
+const schema= require('./schema');
+//const resolvers = require('./resolvers');
 
 
+const neo4j = require('neo4j-driver').v1;
+
+
+let driver = neo4j.driver(
+  'bolt://107.170.69.23:7687',
+  neo4j.auth.basic('graphql', 'graphql')
```

Comment:
Please parameterize credentials and give a quick and easy way how to set them without changing the code.